app/models/Attempt.scala

```diff
@@ -81,6 +81,10 @@ object Attempt {
     Attempt(future recover recovery)
   }
 
+  def fromFuture2[A](future: Future[A])(recovery: PartialFunction[Throwable, Either[AMIableErrors, A]])(implicit ec: ExecutionContext): Attempt[A] = {
```

Comment:
`fromFuture2` probably isn't the right name! `fromFuture` is fine - they have different signatures. We discussed in person, the other `fromFuture` looks a bit weird to me. I think this one is much better so I'll think about migrating the existing usages to this at some point.